### DISCUSSION TITLE
[Question]: 有demo吗

### DISCUSSION THREAD

**Discussion by shentong0722** [danny-avila/LibreChat#1525] [Q&A]
### What is your question?

怎么连个demo图都不放？有人给张搭好的截图呗

### More Details

来张截图

### What is the main subject of your question?

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SailFlorve** [comment]
https://docs.librechat.ai/index.html
Isn't there a YouTube video?
